openedx.yaml

```diff
@@ -0,0 +1,26 @@
+# openedx.yaml
+
+---
+owner:
+    type: team
+    team: edx_data_engineering
+nick: dbt-schema-builder
+tags:
+    - tools
+oeps:
+    oep-2: True
+    oep-3:
+        state: False
+        reason: TODO - Implement for this application if appropriate
+    oep-5:
+        state: False
+        reason: TODO - Implement for this application if appropriate
+    oep-18: True
+    oep-30:
+        state: True
+openedx-release:
+    # The openedx-release key is described in OEP-10:
+    #   https://open-edx-proposals.readthedocs.io/en/latest/oep-0010-proc-openedx-releases.html
+    # The FAQ might also be helpful: https://openedx.atlassian.net/wiki/spaces/COMM/pages/1331268879/Open+edX+Release+FAQ
+    maybe: true   # Delete this "maybe" line when you have decided about Open edX inclusion.
```

Comment:
Ideally you would edit this line.....

Reply:
Will do. Confluence search was down so I couldn't find the new repo checklist. I'll make sure it's on there, and make a note to add it to the cookiecutter setup as well.